modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinSpringServerCodegen.java

```diff
@@ -797,6 +803,20 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
         // TODO: Handle tags
     }
 
+    private boolean containsEnums() {
```

Comment:
thanks for the pr

what about moving this function to ModelUtils class (assuming no similar function exists in that class) ?

Reply:
Done.  I also changed the Java Spring generator to use the `containsEnums` method in ModelUtils.